test/test.coffee

```diff
@@ -508,8 +539,10 @@ describe "StoreIt!", ->
             service.off("added", @publishAdded)
 
         describe "when calling load", ->
-            beforeEach ->
-                service.load()
+            beforeEach (done)->
```

Comment:
Line contains inconsistent indentation<br>Function arrows (-> and =>) must be spaced properly